docs/machine-learning/how-to-guides/explain-machine-learning-model-permutation-feature-importance-ml-net.md

```diff
@@ -0,0 +1,159 @@
+---
+title: "How-To: Explain ML.NET Machine Learning Model Using Permutation Feature Importance"
```

Comment:
I think we could simplify this title. Let's think about the intent of the article.

Reply:
#resolved